docs/index.asciidoc

```diff
@@ -134,6 +167,172 @@ example:
 
 tag events with `_avroparsefailure` when decode fails
 
+[id="plugins-{type}s-{plugin}-ssl_certificate"]
+===== `ssl_certificate`
+
+* Value type is <<path,path>>
+* There is no default value for this setting.
+
+Path to PEM encoded certificate file for client authentication (mutual TLS).
+You may use this setting or <<plugins-{type}s-{plugin}-ssl_keystore_path>>, but not both simultaneously.
+
+*Example*
+[source,ruby]
+----------------------------------
+ssl_certificate => "/path/to/client.crt"
+----------------------------------
+
+[id="plugins-{type}s-{plugin}-ssl_certificate_authorities"]
+===== `ssl_certificate_authorities`
+
+* Value type is a list of <<path,path>>
+* There is no default value for this setting.
+
+Path to PEM encoded CA certificate file(s) for server verification.
+This is an alternative to using <<plugins-{type}s-{plugin}-ssl_truststore_path>>.
+You cannot use this setting and <<plugins-{type}s-{plugin}-ssl_truststore_path>> at the same time.
```

Comment:
You may use this setting or <<plugins-{type}s-{plugin}-ssl_truststore_path>>, but not both simultaneously.